services/reproducible-central/reproducible-central.service.js

```diff
@@ -0,0 +1,91 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  color: Joi.string().required(),
+  message: Joi.string().required(),
+}).required()
+
+export default class ReproducibleCentral extends BaseJsonService {
+  static category = 'dependencies'
+
+  static route = {
+    base: 'reproducible-central/a',
+    pattern: ':groupId/:artifactId/:version',
+  }
+
+  static openApi = {
+    '/reproducible-central/a/{groupId}/{artifactId}/{version}': {
+      get: {
+        summary: 'Reproducible Central',
+        parameters: pathParams(
+          {
+            name: 'groupId',
+            example: 'org.apache.maven',
+          },
+          {
+            name: 'artifactId',
+            example: 'maven-core',
+          },
+          {
+            name: 'version',
+            example: '3.9.9',
+          },
+        ),
+      },
+    },
+  }
+
+  static defaultBadgeData = {
+    label: 'Reproducible Builds',
```

Comment:
From https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines

> The left-hand side of a badge should not advertise. It should be a lowercase noun succinctly describing the meaning of the right-hand side

It is hard to make a suggestion for this because I'm still not really sure what the score means, but basically the default label should describe the meaning of this metric rather than the service that produced it. Although if users want to customise the label, they can.

Reply:
I could say "reproducible builds" to describe https://reproducible-builds.org/, if you really prefer: I think people would still understand

Reply:
OK. Based on your description

> the build has 13 output files, 10 have been successfully reproduced

The other suggestions we could consider here would be "reproducible files" or "files reproduced"